src/foam/box/SessionServerBox.java

```diff
@@ -94,7 +94,8 @@ public void send(Message msg) {
       session.touch();
 
       // TODO: Shouldn't this go somewhere else?
-      if ( req != null && ! SafetyUtil.isEmpty(req.getRequestURI()) ) {
+      if ( effectiveContext.get("group") == null &&
```

Comment:
Why do you want to make this change? Wouldn't it be appropriate to do it even when the appConfig has been determined by the group?

Reply:
I'll add some comments, but when the user was logged in, the group refined appConfig was being replaced.